### PR TITLE
Do not create an egl surface if the window provided is invalid

### PR DESCRIPTION
**Pull request by emmauss**
## What does the pull request do?
Prevents spamming debug output with egl errors on createWindowSurface. This is caused by attempting to create a window surface when the window passed is invalid, ie is 0.

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

### CODE REVIEW

File: src/Avalonia.OpenGL/Egl/EglPlatformOpenGlInterface.cs [hunk 1]
```diff
@@ -50,6 +50,8 @@ public EglSurface CreateWindowSurface(IntPtr window)
         {
             using (PrimaryContext.MakeCurrent())
             {
+                if(window == IntPtr.Zero)
```
Comment (danwalmsley) [inline src/Avalonia.OpenGL/Egl/EglPlatformOpenGlInterface.cs:53]: maybe better to do before makecurrent?

**Review by danwalmsley — CHANGES_REQUESTED** [review]

File: src/Avalonia.OpenGL/Egl/EglPlatformOpenGlInterface.cs (hunk 1 above)
Comment (emmauss) [reply src/Avalonia.OpenGL/Egl/EglPlatformOpenGlInterface.cs]: Yeah. Moved it.

**Review by danwalmsley — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0026015-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0026028-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Pull request merged**